### DISCUSSION TITLE
👉Embed pdf Viewer into Markdown

### DISCUSSION THREAD

**Discussion by abhinit21** [community/community#37607] [Other Feature Feedback, Questions, & Ideas]
How can I insert pdf preview into markdown ?

I tried doing this
```
<embed src="https://raw.githubusercontent.com/abhinit21/Data-Analysis-PySpark/blob/main/Assignment%20PySpark.pdf" />
```

How to get link to raw file is it using `raw.githubusercontent` inplace of `github` ?

**Comment by ossanna16** [comment]
Hi there @abhinit21 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
